Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -1150,6 +1151,136 @@ Reading 2.390625MB ...
 
 To learn more, please refer to the [r2frida wiki](https://github.com/enovella/r2frida-wiki/blob/master/README.md "r2frida Wiki").
 
+##### Information Gathering
```

Comment:
Can you move this section to line 716 instead? We should describe first how to get the names of classes/methods before we are describing "Method hooking".